src/swap/server/volume_contest.py

```diff
@@ -0,0 +1,105 @@
+from decimal import Decimal
+from typing import Optional
+from dataclasses import dataclass, field
+from datetime import datetime, date, timezone, timedelta
+
+import strawberry
+from pymongo.database import Database
+from strawberry.types import Info
+
+
+STABLE_PAIRS = {
```

Comment:
Update ELIGIBLE_PAIRS and STABLE_PAIRS to include all required pairs.

Reply:
updated